webrtc.html

```diff
@@ -3920,8 +3887,10 @@ <h4 id="sec.identity-proxy-communications">Identity Provider
           the IdP provided shall be used.</li>
 
           <li>If the <code>setIdentityProvider()</code> method has not been
-          called, then the browser can use an IdP configured into the
-          browser.</li>
+          called, then the user agent MAY use an IdP configured into the
+          browser.  However, a user agent-provided IdP MUST NOT cause the
+          <code>createOffer</code> or <code>createAnswer</code> invocation to
```

Comment:
Removed this.  I was under some mistaken impression that this would break something.

What does break is if the user agent-provided IdP asks for login and the site doesn't handle the IdpLoginError correctly.  That's just a consideration for the user agent though.